scripts/core/editor3/store/index.ts

```diff
@@ -242,10 +298,14 @@ export function getInitialContent(props) {
     // support standalone instance of editor3 which is not connected to item field
     if (props.editorState != null) {
         var contentState = convertFromRaw(
-            (props.editorState instanceof Array) ? props.editorState[0] : props.editorState,
+            props.editorState instanceof Array
+                ? props.editorState[0]
+                : props.editorState,
         );
 
-        return initializeHighlights(EditorState.createWithContent(contentState)).getCurrentContent();
+        return initializeHighlights(
```

Comment:
It'd be good not to reformat the code that doesn't violate our code style configuration.

Reply:
yeah I don't know why I pushed this, I used an auto-formatter for my new code and I probably included this in